src/test/java/ScheduleTest.java

```diff
@@ -64,35 +65,36 @@ public void testAddLectures() {
     @Test(timeout = 50)
     public void testAddLecturesSingleBefore() {
 
-        schedule = new Schedule();
-        schedule.addLecture(lec1);
-        ArrayList<Section> expected = new ArrayList<>(List.of(lec1, lec2));
+        Schedule schedule = new Schedule();
+        schedule.addLecture(lec1Section);
 
-        schedule.addLecture(lec2);
+        ArrayList<Section> expected = new ArrayList<>(List.of(lec1Section, lec2Section));
 
+        schedule.addLecture(lec2Section);
         assertEquals(expected, schedule.getLectures());
     }
 
     @Test(timeout = 50)
     public void testAddLecturesSingleAfter() {
 
-        schedule = new Schedule();
-        schedule.addLecture(lec2);
-        ArrayList<Section> expected = new ArrayList<>(List.of(lec1, lec2));
-
-        schedule.addLecture(lec1);
```

Comment:
not sure ab this either just looks like lec2 is added before lec1 rather than lec1 before lec2